app/celery/process_ses_receipts_tasks.py

```diff
@@ -77,6 +78,11 @@ def process_ses_results(self, response):
 
         statsd_client.incr("callback.ses.{}".format(notification_status))
 
+        # Record bounces and notifications in Redis
+        if notification_status == NOTIFICATION_PERMANENT_FAILURE:
+            bounce_rate_client.set_sliding_hard_bounce(notification.service_id)
+        bounce_rate_client.set_sliding_notifications(notification.service_id)
```

Comment:
What does this second call do? 🤔

Reply:
The second call adds to the total notifications for that `service_id` regardless of whether it is a hard bounce or not.